geometry_msgs/Point32[] points
std_msgs/String state_msg
int64[] cluster_pos
int64[] point_pos
time segment_stamp